debian/changelog

```diff
@@ -1,3 +1,9 @@
+system76-driver (18.10.5~~alpha) cosmic; urgency=low
+
+  * Add galp3-c
```

Comment:
The changelog message about DAILY WIP blah bah needs to be the first line. You can grab it from the bump version commit